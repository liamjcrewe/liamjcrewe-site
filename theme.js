const theme = {
  colors: {
    primary: '#000000',
    secondary: '#ffffff',
    tertiary: '#08c4e6'
  }
}

export default theme
