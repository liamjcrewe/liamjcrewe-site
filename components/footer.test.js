import { render } from '@testing-library/react'
import { ThemeProvider } from 'styled-components'

import 'jest-styled-components'

import theme from '../theme'
import Footer from './footer'

describe('Footer component', () => {
  it('renders and matches snapshot', () => {
    const { container } = render(
      <ThemeProvider theme={theme}>
        <Footer />
      </ThemeProvider>
    )

    expect(container).toMatchSnapshot()
  })
})
