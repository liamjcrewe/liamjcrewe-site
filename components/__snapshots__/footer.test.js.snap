// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component renders and matches snapshot 1`] = `
.c0 {
  min-height: 20vh;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

.c1 {
  color: #08c4e6;
  -webkit-transition: 0.3s;
  transition: 0.3s;
}

.c1:hover {
  color: #ffffff;
}

<div>
  <div
    class="c0"
  >
    <a
      href="https://github.com/liamjcrewe"
      target="_blank"
    >
      <span
        class="c1"
      >
        <svg
          fill="currentColor"
          height="32"
          size="32"
          stroke="currentColor"
          stroke-width="0"
          viewBox="0 0 16 16"
          width="32"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M8 0C3.58 0 0 3.58 0 8c0 3.54 2.29 6.53 5.47 7.59.4.07.55-.17.55-.38 0-.19-.01-.82-.01-1.49-2.01.37-2.53-.49-2.69-.94-.09-.23-.48-.94-.82-1.13-.28-.15-.68-.52-.01-.53.63-.01 1.08.58 1.23.82.72 1.21 1.87.87 2.33.66.07-.52.28-.87.51-1.07-1.78-.2-3.64-.89-3.64-3.95 0-.87.31-1.59.82-2.15-.08-.2-.36-1.02.08-2.12 0 0 .67-.21 2.2.82.64-.18 1.32-.27 2-.27.68 0 1.36.09 2 .27 1.53-1.04 2.2-.82 2.2-.82.44 1.1.16 1.92.08 2.12.51.56.82 1.27.82 2.15 0 3.07-1.87 3.75-3.65 3.95.29.25.54.73.54 1.48 0 1.07-.01 1.93-.01 2.2 0 .21.15.46.55.38A8.013 8.013 0 0 0 16 8c0-4.42-3.58-8-8-8z"
            fill-rule="evenodd"
          />
        </svg>
      </span>
    </a>
  </div>
</div>
`;
